db/migrate/20200108232831_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration[5.2]
+  def change
+    create_table :movies do |t|
+      t.string :original_title, null: false
```

Comment:
Do we need to call this attribute `original_...` without currently having other titles?

Maybe we should start with `title`.

Reply:
I think we do, yes. It's more or less an industry standard — and it doesn't add any complexity, so why postpone it and risk having to change a column name and code?